requirements.in

```diff
@@ -1,4 +1,5 @@
 # Pre-commit
+coverage
```

Comment:
An aside is I like `pytest-cov` over `coverage` directly these days, because it works with `pytest-xdist`